freva-rest/src/freva_rest/databrowser_api/endpoints.py

```diff
@@ -320,10 +320,12 @@ async def extended_search(
         zarr_stream
         and current_user is None
     ):
-        return JSONResponse(
-            content={"detail": "Not authenticated"},
-            status_code=status.HTTP_401_UNAUTHORIZED
+        logger.exception("User not authenticated for zarr stream.")
```

Comment:
Nit-Pick: This can be an error or warning, we don't really need a traceback. Other than that. LGTM